runtime/mandala/src/lib.rs

```diff
@@ -789,7 +789,8 @@ parameter_type_with_key! {
 				TokenSymbol::ACA |
 				TokenSymbol::KINT |
 				TokenSymbol::KBTC |
-				TokenSymbol::CASH => Balance::max_value() // unsupported
+				TokenSymbol::TAI |
+				TokenSymbol::CASH => 10 * millicent(*currency_id)
```

Comment:
```suggestion
				TokenSymbol::CASH => Balance::max_value() // unsupported
```

move `TokenSymbol::TAI` below `TokenSymbol::PHA`